ready to be merged - MTA-4588: Duplicate entries are appearing because some files inside the target folder are being analyzed (#84)
migtools/mta-documentation: docs/topics/ref_known-issues-7-2-0.adoc

The report generated by containerless CLI analysis is inconsistent when it has analyzed two binary applications that are located in the same folder. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4484[(MTA-4484)]

.Duplicate entries are appearing because some files inside the target folder are being analyzed

> You can see duplicate entries in the built-in provider searches if the application is compiled using the `mvn clean` package. This issue is planned to be resolved in a later release of {ProductShortName}.

Reviewer: using the `mvn clean package`

Reviewer: You can link here to https://issues.redhat.com/browse/MTA-4260 Jira story.
(I've changed this from a doc bug into a feature story).

> You can see duplicate entries in the built-in provider searches if the application is compiled using the `mvn clean` package. This issue is planned to be resolved in a later release of {ProductShortName}.

Author: ```suggestion
You can see duplicate entries in the built-in provider searches if the application is compiled using the `mvn clean package`. This issue is planned to be resolved in a later release of {ProductShortName}. link:https://issues.redhat.com/browse/MTA-4260[(MTA-4260)]
```

> When analyzing a source code project with *Source only*, duplicate entries appear. This issue is caused by the results of running a Maven compilation included in the analysis. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4260[(MTA-4260)]

Author: ```suggestion
When analyzing a source code project with *Source only*, duplicate entries appear. This issue is caused by the results of running a Maven compilation included in the analysis. This issue is planned to be resolved in a later release of {ProductShortName}. link:https://issues.redhat.com/browse/MTA-4260[(MTA-4260)]
```

> When analyzing a source code project with *Source only*, duplicate entries appear. This issue is caused by the results of running a Maven compilation included in the analysis. This issue is planned to be resolved in a later release of {ProductShortName}. link:https://issues.redhat.com/browse/MTA-4260[(MTA-4260)]

Author: ```suggestion
When analyzing a source code project with *Source only*, duplicate entries appear. This issue is caused by the results of running a Maven compilation included in the analysis. This issue is planned to be resolved in a later release of {ProductShortName}.
